roles/configuration-perun/tasks/main.yml

```diff
@@ -36,18 +28,18 @@
     state: directory
     mode: 0755
 
-- name: Create file "{{ log4j_dir }}"
+- name: Create file "{{ logback_dir }}"
   template:
-    src: "{{ log4j_template }}"
-    dest: "{{ log4j_dir }}"
+    src: "{{ logback_template }}"
+    dest: "{{ logback_dir }}"
     owner: "{{ config_files_owner }}"
     group: "{{ perun_group }}"
     mode: 0644
```

Comment:
Since we added tomcat7 to perun group, it would be convenient, if perun user could edit (own) config files. Now they are editable by tomcat7 only. We can keep permissions same, just change ``{{ config_files_owner }}`` default value from tomcat7 to perun.
We want to switch to user tomcat later, so if property is unused anywhere else, we could replace ``{{ config_files_owner }}`` with ``{{ perun_login }}``.